### PR TITLE
SNAP-1494: Dashboard shows an exception stack trace when a server goes down

### PR DESCRIPTION
**Pull request by snappy-sachin**
## Changes proposed in this pull request

 - Handle exception and log into log file.

## Patch testing

 - Tested Manually

### CODE REVIEW

**Comment by snappy-sachin** [comment]
Run precheckin

File: core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala [hunk 1]
```diff
@@ -167,12 +167,17 @@ object SnappyEmbeddedTableStatsProviderService extends TableStatsProviderService
...
+      if (dataServers != null && dataServers.size() > 0) {
+        result = FunctionService.onMembers(dataServers)
+            .withCollector(new GfxdListResultCollector())
+            .execute(SnappyRegionStatsCollectorFunction.ID).getResult().
+            asInstanceOf[java.util.ArrayList[SnappyRegionStatsCollectorResult]]
+            .asScala
+      }
+    }
+    catch {
+      case e: Exception => log.warn(e.getMessage, e)
```
Comment (sumwale) [inline core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala]: change to: case NonFatal(e)

**Comment by snappy-sachin** [comment]
Ran pre checkin on aws instance. All tests passed except the known failures.

File: core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala (hunk 1 above)
Comment (snappy-sachin) [inline core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala]: Missed while merging. 
Added JIRA bug (SNAP-1596) for tracking it.

**Pull request merged**